stubs/ibm-db/ibm_db.pyi

```diff
@@ -0,0 +1,233 @@
+SQL_ATTR_TRUSTED_CONTEXT_USERID: int
+SQL_ATTR_USE_TRUSTED_CONTEXT: int
+SQL_ATTR_XML_DECLARATION: int
+SQL_AUTOCOMMIT_OFF: int
+SQL_AUTOCOMMIT_ON: int
+SQL_BIGINT: int
+SQL_BINARY: int
+SQL_BIT: int
+SQL_BLOB: int
+SQL_BLOB_LOCATOR: int
+SQL_BOOLEAN: int
+SQL_CHAR: int
+SQL_CLOB: int
+SQL_CLOB_LOCATOR: int
+SQL_CURSOR_DYNAMIC: int
+SQL_CURSOR_FORWARD_ONLY: int
+SQL_CURSOR_KEYSET_DRIVEN: int
+SQL_CURSOR_STATIC: int
+SQL_DBCLOB: int
+SQL_DBCLOB_LOCATOR: int
+SQL_DBMS_NAME: int
+SQL_DBMS_VER: int
+SQL_DECFLOAT: int
+SQL_DECIMAL: int
+SQL_DOUBLE: int
+SQL_FALSE: int
+SQL_FLOAT: int
+SQL_GRAPHIC: int
+SQL_INDEX_CLUSTERED: int
+SQL_INDEX_OTHER: int
+SQL_INTEGER: int
+SQL_LONGVARBINARY: int
+SQL_LONGVARCHAR: int
+SQL_LONGVARGRAPHIC: int
+SQL_NUMERIC: int
+SQL_PARAM_BIND_BY_COLUMN: int
+SQL_PARAM_INPUT: int
+SQL_PARAM_INPUT_OUTPUT: int
+SQL_PARAM_OUTPUT: int
+SQL_REAL: int
+SQL_ROWCOUNT_PREFETCH_OFF: int
+SQL_ROWCOUNT_PREFETCH_ON: int
+SQL_SMALLINT: int
+SQL_TABLE_STAT: int
+SQL_TINYINT: int
+SQL_TRUE: int
+SQL_TYPE_DATE: int
+SQL_TYPE_TIME: int
+SQL_TYPE_TIMESTAMP: int
+SQL_VARBINARY: int
+SQL_VARCHAR: int
+SQL_VARGRAPHIC: int
+SQL_WCHAR: int
+SQL_WLONGVARCHAR: int
+SQL_WVARCHAR: int
+SQL_XML: int
+USE_WCHAR: int
+WCHAR_NO: int
+WCHAR_YES: int
+
+@final
+class IBM_DBClientInfo:
+    def __new__(cls: type[Self], *args: object, **kwargs: object) -> Self: ...
+    APPL_CODEPAGE: str
+    CONN_CODEPAGE: str
+    DATA_SOURCE_NAME: str
+    DRIVER_NAME: str
+    DRIVER_ODBC_VER: str
+    DRIVER_VER: str
+    ODBC_SQL_CONFORMANCE: str
+    ODBC_VER: str
+
+@final
+class IBM_DBConnection:
+    def __new__(cls: type[Self], *args: object, **kwargs: object) -> Self: ...
+
+@final
+class IBM_DBServerInfo:
+    def __new__(cls: type[Self], *args: object, **kwargs: object) -> Self: ...
+    DBMS_NAME: str
+    DBMS_VER: str
+    DB_CODEPAGE: str
+    DB_NAME: str
+    DFT_ISOLATION: str
+    IDENTIFIER_QUOTE_CHAR: str
+    INST_NAME: str
+    ISOLATION_OPTION: str
+    KEYWORDS: str
+    LIKE_ESCAPE_CLAUSE: bool
+    MAX_COL_NAME_LEN: bytes
+    MAX_IDENTIFIER_LEN: str
+    MAX_INDEX_SIZE: bytes
+    MAX_PROC_NAME_LEN: bytes
+    MAX_ROW_SIZE: bytes
+    MAX_SCHEMA_NAME_LEN: bytes
+    MAX_STATEMENT_LEN: bytes
+    MAX_TABLE_NAME_LEN: bytes
+    NON_NULLABLE_COLUMNS: str
+    PROCEDURES: bool
+    SPECIAL_CHARS: str
+    SQL_CONFORMANCE: str
+
+@final
+class IBM_DBStatement:
+    def __new__(cls: type[Self], *args: object, **kwargs: object) -> Self: ...
+
+def active(connection: IBM_DBConnection) -> bool: ...
+def autocommit(connection: IBM_DBConnection, value: int | None = ...) -> int | bool: ...
+def bind_param(
+    stmt: IBM_DBStatement,
+    parameter_number: int,
+    variable: str,
+    parameter_type: int | None = ...,
+    data_type: int | None = ...,
+    precision: int | None = ...,
+    scale: int | None = ...,
+    size: int | None = ...,
+) -> bool: ...
+def callproc(
+    connection: IBM_DBConnection, procname: str, parameters: tuple[object, ...]
```

Comment:
```suggestion
    connection: IBM_DBConnection | None, procname: str, parameters: tuple[object, ...] = ...
```
If I'm reading this correctly it returns an `IBM_DBStatement` if parameters is given (and non-empty) and a tuple otherwise. This could be expressed with overloads.